### DISCUSSION TITLE
Does OpenMetadata support new types as configuration?

### DISCUSSION THREAD

**Discussion by gdiwakarlr** [open-metadata/OpenMetadata#7916] [Q&A]
Hello,

We r exploring Open Source Data Catalogs... OpenMetadata looks amazing.

Question: does OpenMetadata support new types as configuration?
(I searched existing discussions, some have partial info - but none r clear on what ur stance is, hence asking)

Details:
- Just like how `Tables` is an entity type supported in the Catalog; can we add others like say `File`, `View`, `Report` or any others specific to a company's domain?
- e.g. an insurance company may have types like `Policy`, `Report` etc. Or say a finance company may have `Commodity`, `Portfolio`, `Asset-class` etc.
- If so, does that need us (OpenMetadata adopters) to fork the repo & create our own schemas + code changes or is this something supported as configuration?

Thank u.

**Comment by harshach** [comment]
Hi @gdiwakarlr, 
Thanks for your interest in OpenMetadata.
Our goal in OpenMetadata is to standardize entities if there is something is not already supported, please let us know we will add it. Each entity has an option to add custom attributes. One thing we feel strongly about is existing and older generations of metadata platforms expose key/value pairs which will cause all kinds of integration issues hence the goal of clear specification of the entity. Example Policy and Report is already in OpenMetadata we would like to understand how we can model Commodity, Portfolio, and Asset-class and add in OpenMetadata.
We are most active in slack, please reach out to us there https://slack.open-metadata.org
